tests/zipfile.cpp

```diff
@@ -187,20 +188,73 @@ SCENARIO("ZipFile with a valid zip archive", "[ZipFile] [FileCollection]")
                 REQUIRE_FALSE(zf.getInputStream("inexistant", zipios::FileCollection::MatchPath::MATCH));
                 REQUIRE_FALSE(zf.getInputStream("inexistant", zipios::FileCollection::MatchPath::IGNORE));
                 REQUIRE(zf.getName() == "tree.zip");
-                REQUIRE(zf.size() == tree.size());
                 zf.mustBeValid(); // not throwing
-
+                if (zf.size() != tree.size())
```

Comment:
Debug code.
On windows these are not equal. tree.size() is larger than zf.size()
Seems it it the directory entries that are missing:
```
tree/f1
tree/f2
tree/d1/f4
tree/d1/f5
```
Whereas on linux you'd get

```
tree/f1
tree/f2
tree/d1   <--
tree/d1/f4
tree/d1/f5
```